script.js

```diff
@@ -0,0 +1,51 @@
+// storing HTML elements 
+const first_name = document.querySelector("#first_name");
+const last_name = document.querySelector("#last_name");
+let full_name = document.querySelector("#full_name");
+full_name.value = "";
+let regex = /^[a-zA-Z]+$/;
+first_name.addEventListener('input',()=> {
```

Comment:
It should be written like this ->
`first_name.addEventListener('input', () => {`